Registry/Registry.EM_COMMON

```diff
@@ -901,6 +901,11 @@ state    real   CHC_SFCDIF      ij          misc       1         -     r      "C
 state    real   CMGR_SFCDIF     ij          misc       1         -     r      "CMGR_SFCDIF" "" ""
 state    real   CHGR_SFCDIF     ij          misc       1         -     r      "CHGR_SFCDIF" "" ""
 
+# Eclipse amontornes-bcodina 2015/09
+state    integer   ECMASK          ij          misc        1        -     rh     "ECMASK"  "ECLIPSE MASK"     "dimensionless"
```

Comment:
@irowebbn 
Isaac,
These two 2d-arrays need to be packaged

Reply:
@davegill The data file a text file, about 80 kb in size.

Reply:
Addressed.